### ISSUE TITLE
Compiler isn't collect env.

### ISSUE THREAD

**Issue by blackinno** [liteflow-labs/liteflow-js#164]
The first environment is working but the second environment doesn't work because `0=PROVIDER_ENDPOINT=...` 
You can look at the compile.

**YAML:** 
```yml
name: Aragon
steps:
  - type: trigger
    instance:
      src: ./services/ethereum
      env:
        - PROVIDER_ENDPOINT=https://rinkeby.infura.io/v3/1aeb73da68804152b224a285ef96ce02
    eventKey: log
  - type: filter
    conditions:
      address: '0x7a80f0b3848242281a5ae126596b54d4010c30fe'
      eventSignature: '0x4d72fe0577a3a3f7da968d7b892779dde102519c25527b29cf7054f245c791b9'
  - type: task
    instance:
      src: ./services/ethereum
      env:
        - PROVIDER_ENDPOINT=https://rinkeby.infura.io/v3/1aeb73da68804152b224a285ef96ce02
    taskKey: decodeLog
    inputs:
      eventAbi: '{"anonymous":false,"inputs":[{"indexed":true,"name":"voteId","type":"uint256"},{"indexed":true,"name":"creator","type":"address"},{"indexed":false,"name":"metadata","type":"string"}],"name":"StartVote","type":"event","signature":"0x4d72fe0577a3a3f7da968d7b892779dde102519c25527b29cf7054f245c791b9"}'
      address: { key: address }
      data: { key: data }
      topics: { key: topics }
      logIndex: { key: logIndex }
      transactionHash: { key: transactionHash }
      transactionIndex: { key: transactionIndex }
      blockHash: { key: blockHash }
      blockNumber: { key: blockNumber }
```
**Compile:**
```json
{
  "name": "Aragon",
  "root": {
    "key": "node-0",
    "instance": {
      "service": ...
      "env": [
        "PROVIDER_ENDPOINT=https://rinkeby.infura.io/v3/1aeb73da68804152b224a285ef96ce02"
      ]
    },
    "eventKey": "log",
    "type": "EVENT",
    "children": [
      {
        "key": "node-1",
        "conditions": ...,
        "type": "FILTER",
        "children": [
          {
            "key": "node-2-inputs",
            "outputs": ...,
            "type": "MAP",
            "children": [
              {
                "key": "node-2",
                "instance": {
                  "service": ...,
                  "env": [
                    "0=PROVIDER_ENDPOINT=https://rinkeby.infura.io/v3/1aeb73da68804152b224a285ef96ce02"
                  ]
                },
                "taskKey": "decodeLog",
                "type": "TASK",
                "children": []
              }
            ]
          }
        ]
      }
    ]
  }
}

```
**Database**
<img width="1680" alt="Screen Shot 2563-06-25 at 17 34 27" src="https://user-images.githubusercontent.com/16553272/85704119-37168480-b70a-11ea-8019-351cd2702a4e.png">

**Comment by blackinno** [comment]
**Update**
Also if have many environments in one task, it's saved or compile with 
`0=TWILIO_ACCOUNT_SID=ACdc9875f62ebb6cb7adafca9fc2e29...,1=TWILIO_AUTH_TOKEN=1b334fef563c257b90c0de53095a8...,2=FROM=+1830386...
`

```yml
name: twilio
steps:
  - type: trigger
    endpointKey: twilio
  - type: task
    instance:
      src: /Users/benthotsaphon/Documents/MESG/services/twilio
      env:
        - TWILIO_ACCOUNT_SID=ACdc9875f62ebb6cb7adafca9fc2e29...
        - TWILIO_AUTH_TOKEN=1b334fef563c257b90c0de53095a8...
        - FROM=+1830386....
    taskKey: sms
    inputs:
      to: { key: body.to }
      body: { key: body.message }

```

**Comment by antho1404** [comment]
this should be fixed on #166
